### PR TITLE
Add command to re-import failed error messages

### PR DESCRIPTION
**Pull request by timbussmann**

### CODE REVIEW

**Comment by SzymonPobiega** [comment]
@Particular/servicecontrol-maintainers we have a problem here. The component that re-imports failed audits does not send messages to the `audit.log` queue. Is that behavior intentional? I don't think so. Based on that, the failed error re-importer should probably (apart from actually storing the processing attempt)
 * send the message to `error.log`
 * publish `MessageFailed` even on the bus

Is that correct?

**Comment by boblangley** [comment]
@SzymonPobiega I think you are correct. All import actions should be performed the same as if the satellite was processing the message.

File: src/ServiceControl.AcceptanceTests/Audit/When_a_message_fails_to_import.cs [hunk 1]
```diff
@@ -10,18 +10,29 @@
     using NUnit.Framework;
     using ServiceControl.CompositeViews.Messages;
     using ServiceControl.Operations;
+    using Conventions = NServiceBus.AcceptanceTesting.Customization.Conventions;
 
     class When_a_message_fails_to_import : AcceptanceTest
```
[the remark this answers is not in the document]
Comment (SzymonPobiega) [reply src/ServiceControl.AcceptanceTests/Audit/When_a_message_fails_to_import.cs]: The `AuditIngestor` does not raise any events so nope. That said, this test is good to ensure that the behavior remains correct after switching from `AuditPersister` to `AuditIngestor` but is not so great when it comes to asserting on every side effect that is expected to happen when processing an audit message.

But I think such test is out of scope here.

**Comment by timbussmann** [comment]
build is green now. @SzymonPobiega feel free to merge if you're happy too.

**Pull request merged**